server/handlers.go

```diff
@@ -61,9 +62,9 @@ func (server HTTPServer) getContentForRule(writer http.ResponseWriter, request *
 
 	// check for internal rule permissions
 	if internal := content.IsRuleInternal(ruleID); internal == true {
-		ok := server.checkInternalRulePermissions(writer, request)
+		ok, err := server.checkInternalRulePermissions(request)
 		if ok != true {
```

Comment:
you meant `err != nil`?

Reply:
I was checking the `checkInternalRulePermission` and I realized that returning the `error` will be enough, because the `bool` value is always `true` when `error` is `nil`, and `false` if `error != nil`, so probably I will refactor the method and the callers. Thank you very much!